src/index.html

```diff
@@ -9,6 +9,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <main class = "main">
+      <header class = "header">
+        <a href="index.html">
+          <img src="./images/logo.png" 
+          alt="MOYO logotype" 
+          class = "MOYOimg">
+        </a>
+        <nav class = "nav">
+          <ul class = "nav_list">
+            <li class = "nav_item">
+              <a href="#apple" class = "nav_links is-active">Apple</a>
+            </li>
+
+            <li class = "nav_item">
+              <a href="#samsung" class = "nav_links">samsung</a>
+            </li>
```

Comment:
It's not nav_links, it's one nav_link, change in all places
```suggestion
            <li class = "nav_item">
              <a href="#samsung" class = "nav_link">samsung</a>
            </li>
```